eiffel-vocabulary/EiffelArtifactDeployedEvent.md

```diff
@@ -19,7 +19,31 @@
 The EiffelArtifactDeployedEvent id to declare that an artifact has been deployed into an environment.
 
 ## Data Members
- 
+### data.location
+__Type:__ String  
+__Required:__ No  
+__Description:__ The location of the deployed artifact.
```

Comment:
What format for "location"?

Reply:
We are considering that this "location" is probably not really necessary.
The UC is not really clear "Why" we would need to know the location and publish it considering that a Activity execution (representing the deployment) would already know this.

Proposal to remove data.location from the EiffelArtifactDeployedEvent ?